### PR TITLE
Added scene and transition related concepts.

### PR DESCRIPTION
**Pull request by mpomarlan**

### CODE REVIEW

File: owl/EASE-SAY.owl [hunk 1]
```diff
@@ -185,6 +208,83 @@ Note that in a semantic sense such clauses always request information, but in a
             </owl:Restriction>
         </rdfs:subClassOf>
     </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE-SAY.owl#Scene -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-SAY.owl#Scene">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Situation"/>
```
Comment (daniel86) [inline owl/EASE-SAY.owl:217]: a comment is missing here

File: owl/EASE-SAY.owl [hunk 2]
```diff
@@ -185,6 +208,83 @@ Note that in a semantic sense such clauses always request information, but in a
...
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE-SAY.owl#StateTransition -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-SAY.owl#StateTransition">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Transition"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ease-crc.org/ont/EASE.owl#hasInitialState"/>
```
Comment (daniel86) [inline owl/EASE-SAY.owl:245]: the relation name seems confusing as there are State events. Why not hasInitialScene?

File: owl/EASE-SAY.owl [hunk 3]
```diff
@@ -185,6 +208,83 @@ Note that in a semantic sense such clauses always request information, but in a
...
+                <owl:someValuesFrom rdf:resource="http://www.ease-crc.org/ont/EASE-SAY.owl#Scene"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ease-crc.org/ont/EASE.owl#hasTerminalState"/>
```
Comment (daniel86) [inline owl/EASE-SAY.owl:251]: same as above

File: owl/EASE-SAY.owl [hunk 4]
```diff
@@ -185,6 +208,83 @@ Note that in a semantic sense such clauses always request information, but in a
...
+        <rdfs:comment>A theory used to describe, analyze, and reason with the meaning of a linguistic message.
+
+Note that such theories are useful both when analyzing an actual linguistic production, and when creating a linguistic production to describe some observed experience.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE-SAY.owl#StateTransition -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-SAY.owl#StateTransition">
```
Comment (daniel86) [inline owl/EASE-SAY.owl:241]: the axiomatization seems problematic to me, as no relation between initial/terminal scene and the corresponding states is given.
Maybe better leave out the includesEvent axioms?

File: owl/EASE-SAY.owl [hunk 5]
```diff
@@ -185,6 +208,83 @@ Note that in a semantic sense such clauses always request information, but in a
...
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#includesEvent"/>
+                <owl:minQualifiedCardinality rdf:datatype="http://www.w3.org/2001/XMLSchema#nonNegativeInteger">2</owl:minQualifiedCardinality>
+                <owl:onClass rdf:resource="http://www.ease-crc.org/ont/EASE.owl#State"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:comment>A transition between two states brought about by the Action of some Agent.</rdfs:comment>
```
Comment (daniel86) [inline owl/EASE-SAY.owl:286]: probably the comment should mention scenes

File: owl/EASE-SAY.owl (hunk 4 above)
Comment (mpomarlan) [reply owl/EASE-SAY.owl:241]: The ones we inherit from DUL will stay, but I can remove the other ones.

File: owl/EASE-SAY.owl [hunk 6]
```diff
@@ -15,6 +15,41 @@
...
+    //
+    ///////////////////////////////////////////////////////////////////////////////////////
+     -->
+
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE-SAY.owl#hasInitialScene -->
+
+    <owl:ObjectProperty rdf:about="http://www.ease-crc.org/ont/EASE-SAY.owl#hasInitialScene">
```
Comment (daniel86) [inline owl/EASE-SAY.owl:31]: comment missing

File: owl/EASE-SAY.owl [hunk 7]
```diff
@@ -15,6 +15,41 @@
...
+        <rdf:type rdf:resource="http://www.w3.org/2002/07/owl#IrreflexiveProperty"/>
+        <rdfs:domain rdf:resource="http://www.ease-crc.org/ont/EASE-SAY.owl#StateTransition"/>
+        <rdfs:range rdf:resource="http://www.ease-crc.org/ont/EASE-SAY.owl#Scene"/>
+    </owl:ObjectProperty>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE-SAY.owl#hasTerminalScene -->
+
+    <owl:ObjectProperty rdf:about="http://www.ease-crc.org/ont/EASE-SAY.owl#hasTerminalScene">
```
Comment (daniel86) [inline owl/EASE-SAY.owl:43]: comment missing

**Pull request merged**